02/studenttests.ml

```diff
@@ -256,13 +261,25 @@ let cc_sub = fun (dest:Int64.t) (src:Int64.t) -> Gradedtests.test_machine
   ;InsB0 (Subq, [Imm (Lit src); ~%Rax]);InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag
   ]
 
+let cc_imul = fun (a:int64) (b:int64) -> Gradedtests.test_machine
+  [InsB0 (Movq, [Imm (Lit a); ~%Rax]);InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag
+  ;InsB0 (Imulq, [Imm (Lit b); ~%Rax]);InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag
+  ]
+
+let cc_neg = fun (a:int64) -> Gradedtests.test_machine
+  [InsB0 (Movq, [Imm (Lit a); ~%Rax]);InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag
+  ;InsB0 (Negq, [~%Rax]);InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag;InsFrag
+  ]
+
 let condition_flag_set_tests = 
   [ (* Logic instruction. *)
     ("cc_and", Gradedtests.cs_test 2 (cc_and ()) (false, false, false))
   ; ("cc_or",  Gradedtests.cs_test 2 (cc_or  ()) (false, false, false))
   ; ("cc_xor", Gradedtests.cs_test 2 (cc_xor ()) (false, false, false))
   ; ("cc_not", Gradedtests.csi_test 2 (cc_not ()))
-  ; ("cc_not_negative", Gradedtests.cs_test 3 (cc_not_negative ()) (false, false, false))
```

Comment:
Why are you removing this test? Is this covered by a new one? If this is the case please remove also the `cc_not_negative` definition

Reply:
Please use cc as prefix name, as all the other cases